onebusaway-android/src/main/java/org/onebusaway/android/report/ui/SimpleArrivalListFragment.java

```diff
@@ -273,6 +279,6 @@ private ArrivalsListLoader getArrivalsLoader() {
     }
 
     public void setCallback(Callback callback) {
-        this.callback = callback;
+        this.mCallback = callback;
```

Comment:
Sorry, to clarify - you should be able to change this line to:
`mCallback = callback;`

So, just remove the reference to "this" instance of the class.